src/element-observation.js

```diff
@@ -63,7 +63,7 @@ export class StyleObserver {
   _setProperty(style, value) {
     let priority = '';
 
-    if (value && value.indexOf('!important') !== -1) {
+    if (typeof value !== typeof undefined && typeof value.indexOf !== typeof undefined && value.indexOf('!important') !== -1) {
```

Comment:
That makes sense and it still works for me, in the midst of rushing to get this fixed, I completely didn't realise that I wasn't doing a proper null check there. I'll update it with that code.